.changeset/add-gujarati-translations.md

```
@@ -0,0 +1,6 @@
+---
+'@c15t/translations': minor
+'@c15t/schema': patch
+---
+
+Add Gujarati (`gu`) translations for cookie banner, consent dialog, and IAB TCF consent UI copy.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)